examples/references.bib

```diff
@@ -338,6 +338,13 @@ @article{nuijten2015default
   year          = {2015},
   publisher     = {Springer}
 }
+@book{rasmussen2003gaussian,
+  title         = {Gaussian Processes for Machine Learning},
+  author        = {Rasmussen, Carl Edward and Williams, Christopher K. I.},
+  year          = {2006},
+  isbn          = {026218253X},
+  publisher     = {The MIT Press}
```

Comment:
I would also add the `url` field with the link in the notebook body: http://gaussianprocess.org/gpml/. After that, other notebooks can cite it and the citation will have everything, making it easier to reference this resource in other or new notebooks